### ISSUE TITLE
Suggestion: Add the possiblity to remove a vote for an idea

### ISSUE THREAD

**Issue by Liebeck** [ivandiazwm/phpback#89]

**Comment by benodilo** [comment]
Good suggestion ! We can add 1,2 ou 3 points but we can't reset it (0 point)

**Comment by ivandiazwm** [comment]
Actually, you can. Just go to the profile page (click on your name when logged) and you'll find it in "Reset votes" tab

**Comment by benodilo** [comment]
In the vote button we need a "0 vote"

Hinojosa Benjamin

**Comment by shackrock** [comment]
Agreed w/ @benodilo - the functionality may exist, just need to add to the UI on all pages for better user interactions.
